SPOCK_SYSTEM_SETUP.md

```
@@ -0,0 +1,598 @@
+| Role | Hostname | Database Container | Port |
+|------|----------|-------------------|------|
+| **PRIMARY** | `beta.brainwires.net` | `supabase-dev-db` | 25432 |
+| **STANDBY** | `brainwires.net` | `supabase-dev-db` | 25432 |
+
+### Connectivity
+
+Cross-machine connectivity is achieved via **Cloudflare Zero Trust tunnels**, providing secure connections without exposing database ports to the public internet.
+
+### Replication Flow
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)